cmd/feed-ingress/main.go

```diff
@@ -65,11 +75,20 @@ func init() {
 		defaultNginxUpdatePeriod                 = time.Second * 30
 		defaultElbLabelValue                     = ""
 		defaultElbDrainDelay                     = time.Second * 60
+		defaultDrainDelay                        = time.Second * 30
```

Comment:
Is there any reason we couldn't rename the `defaultElbDrainDelay` constant, and use that both for ELB and IPVS?

Reply:
done we use drainDelay for both now.